src/main/java/seedu/address/logic/commands/AssignCommand.java

```diff
@@ -49,12 +49,12 @@ public AssignCommand(Index personIndex, Index taskIndex) {
     public CommandResult execute(Model model, OfficeConnectModel officeConnectModel) throws CommandException {
         requireAllNonNull(model, officeConnectModel);
 
-        List<Person> personList = model.getFilteredPersonList();
+        List<Person> personList = model.getAddressBook().getPersonList();
         if (personIndex.getZeroBased() >= personList.size()) {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
 
-        List<Task> taskList = officeConnectModel.getTaskModelManager().getFilteredItemList();
+        List<Task> taskList = officeConnectModel.getTaskModelManager().getReadOnlyRepository().getData();
```

Comment:
For these changes, would assigning the person to task based on the actual task list and person list make it inconsistent with the rest of our command such as `delete`, `deletetask`,  `mark`, etc, which is still currently based on the displayed lists instead of the actual lists. 

What do you think?

Reply:
Right, since `delete`, `mark` commands need specific indexes I think would be better to align with the rest. 

Ill revert the change back to the original one.

Reply:
> Member

Agree with Yiting, too many different command may confuse the user.

However after v1.2, it is clear that we only left a few CRUD features need to implement which is likely to be fast.
Thus we can consider our next focus areas.
Currently my suggestion is either we focus on GUI or the conveniency of commands
Here are the examples:
- GUI
    - We can consider interaction effect such as click effect and animations
- Conveniency 
    - Make the command more natural
           - Instead of `addtask Biking` or `add Peter`  we could use `add task Biking`  or  `add person peter`. 
           - Instead of `review 1` or `reviewtask 1` we could try `review person -i 1` and `review task -i 1 `
           - In addtion to the index review we could also add `review person Peter` or `review task Biking` if there is ambiguity we will display to result and ask the user input the index, thus a multi step command is needed.